### PR TITLE
Add Dec discount offer to hosted header

### PR DESCRIPTION
**Pull request by michaelatmender**
Discount offer should show for hosted Trial users only, until the end of December

### CODE REVIEW

File: src/js/components/settings/settings.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ export const Settings = ({ allowUserManagement, currentUser, hasMultitenancy, hi
 
 const mapStateToProps = state => {
   const currentUser = state.users.byId[state.users.currentUser];
-  const { plan = 'os', trial = false } = state.organization.organization;
+  const plan = 'os';
+  const trial = true;
```
Comment (mzedel) [inline src/js/components/settings/settings.js:110]: this is probably left over I think...

File: src/js/components/settings/settings.js [hunk 2]
```diff
@@ -50,12 +50,12 @@ export const Settings = ({ allowUserManagement, currentUser, hasMultitenancy, hi
...
     upgrade: {
-      admin: false,
-      enterprise: false,
+      admin: true,
+      enterprise: true,
       multitenancy: true,
       trial: true,
-      userManagement: false,
-      component: <Upgrade history={history} />,
+      userManagement: true,
```
Comment (mzedel) [inline src/js/components/settings/settings.js:57]: I could see this only being allowed for admins (though I personally would accept an upgrade from every user 😅) but `enterprise` and `userManagement` are probably not required right?

File: src/less/main.less [hunk 3]
```diff
@@ -21,7 +21,7 @@ body {
   display: grid;
   height: 100%;
   grid-template-columns: 200px 1fr;
-  grid-template-rows: 57px 1fr;
+  grid-template-rows: min-content 1fr;
```
Comment (mzedel) [inline src/less/main.less:24]: very nice 👍 - I still haven't internalised these...

**Review by mzedel — CHANGES_REQUESTED** [review]
nicely done, just two things that might be still in from the development...

File: src/js/components/settings/settings.js (hunk 1 above)
Comment (michaelatmender) [reply src/js/components/settings/settings.js:110]: Yes, this and the previous comment are leftovers. Careless committing from me, sorry!

File: src/js/components/settings/settings.js [hunk 4]
```diff
@@ -106,7 +106,6 @@ export const Settings = ({ allowUserManagement, currentUser, hasMultitenancy, hi
 
 const mapStateToProps = state => {
   const currentUser = state.users.byId[state.users.currentUser];
-  const { plan = 'os', trial = false } = state.organization.organization;
```
Comment (mzedel) [inline src/js/components/settings/settings.js:109]: this is still required 😉

**Review by mzedel — CHANGES_REQUESTED** [review]
that was a tiny bit too much... sorry about the back and forth, I'm a bit too deep in the testing branch right now, otherwise I'd push it myself 😟

**Review by mzedel — APPROVED** [review]
👍

**Comment by michaelatmender** [comment]
@tranchitella do you have the ability to merge this? The ci/gitlab status seems fine on https://gitlab.com/Northern.tech/Mender/gui/-/pipelines/227214309 but is not reporting success here

**Pull request merged**